Libs/MRML/Core/vtkMRMLVolumeNode.cxx

```diff
@@ -669,6 +669,43 @@ const char* vtkMRMLVolumeNode::ComputeScanOrderFromIJKToRAS(vtkMatrix4x4 *ijkToR
   }
 }
 
+//----------------------------------------------------------------------------
+// Checks that a vtkImageData has origin (0,0,0), spacing (1,1,1), and an identity direction matrix
+bool imageDataHasIdentityAffine(vtkImageData *imageData)
```

Comment:
```suggestion
bool HasDefaultSpatialProperties(vtkImageData* imageData)
```

Reply:
I would call this method `VerifyImageDataGeometry` or `IsImageDataGeometryValid`, because we usually refer to image origin, spacing, axis directions as "geometry", and the goal of the check is to verify if it is valid (not that it equals to some default; and not that it is specifically identity - because in the future non-identity matrix will be accepted).